tests/sentry/integrations/test_ticket_rules.py

```diff
@@ -149,3 +150,140 @@ def test_fails_validation(self):
                 },
             )
             assert response.status_code == 400
+
+
+class JiraServerTicketRulesTestCase(RuleTestCase, BaseAPITestCase):
+    rule_cls = JiraServerCreateTicketAction
+    mock_jira = None
+
+    def get_client(self):
+        if not self.mock_jira:
+            self.mock_jira = MockJira()
```

Comment:
You might consider setting up the client like it's done in [this test file](https://github.com/getsentry/sentry/blob/master/tests/sentry/integrations/jira_server/test_client.py#L48) - I'm not sure how accurate a `MockJira` is to Jira Server.

Reply:
Switched to using mock responses. 

The `MockJira` could be used, but its responses would have the wrong URLs in the stub JSON responses like `https://example.com` instead of `https://jira.example.com`. We could switch the URLs to work with it, I don't lean heavily either way.